src/Microsoft.ApplicationInsights.AspNetCore/Implementation/TelemetryConfigurationOptionsSetup.cs

```diff
@@ -45,6 +49,15 @@ public void Configure(TelemetryConfiguration configuration)
                 configuration.InstrumentationKey = this.applicationInsightsServiceOptions.InstrumentationKey;
             }
 
+            if (this.telemetryProcessorFactories.Any())
```

Comment:
BTW, quick pulse telemetry module and telemetry processor need to know about each other. This link between them will not survive thru the re-`Build()`. So `Initialize` on module should be called after `Build` on the chain.

Can this be achieved in this case?

(real solution will be to change the design of quick pulse)